homeassistant/components/manual/alarm_control_panel.py

```diff
@@ -37,16 +36,20 @@
 from homeassistant.helpers.restore_state import RestoreEntity
 import homeassistant.util.dt as dt_util
 
-_LOGGER = logging.getLogger(__name__)
+from .const import (
+    ATTR_NEXT_STATE,
+    ATTR_PREVIOUS_STATE,
+    CONF_CODE_ARM_REQUIRED,
+    CONF_CODE_TEMPLATE,
+    DEFAULT_ALARM_NAME,
+    DEFAULT_ARMING_TIME,
+    DEFAULT_DELAY_TIME,
+    DEFAULT_DISARM_AFTER_TRIGGER,
+    DEFAULT_TRIGGER_TIME,
+)
 
-CONF_CODE_TEMPLATE = "code_template"
-CONF_CODE_ARM_REQUIRED = "code_arm_required"
+_LOGGER = logging.getLogger(__name__)
 
-DEFAULT_ALARM_NAME = "HA Alarm"
-DEFAULT_DELAY_TIME = datetime.timedelta(seconds=60)
-DEFAULT_ARMING_TIME = datetime.timedelta(seconds=60)
-DEFAULT_TRIGGER_TIME = datetime.timedelta(seconds=120)
-DEFAULT_DISARM_AFTER_TRIGGER = False
```

Comment:
Doesn't make much sense I think to move constants to a `const` file if they are only used for the config schema in this file.